apps/sms/views/conversation/index.html

```diff
@@ -0,0 +1,558 @@
+  <script defer src="/shared/js/component_utils.js"></script>
+  <script defer src="/shared/elements/gaia_menu/script.js"></script>
+  <script defer src="/shared/elements/gaia_sim_picker/script.js"></script>
+  <link rel="stylesheet" type="text/css" href="/shared/elements/gaia_menu/style.css">
+  <link rel="stylesheet" type="text/css" href="/shared/elements/gaia_sim_picker/style.css">
+  -->
+  <!-- Shared code -->
+  <script defer src="/shared/js/l10n.js"></script>
+  <script defer src="/shared/js/l10n_date.js"></script>
+  <!-- Should be removed while gecko support navigator.mozHour12 API -->
+  <script defer src="/shared/js/date_time_helper.js"></script>
+  <script defer src="/shared/js/template.js"></script>
+  <script defer src="/shared/js/async_storage.js"></script>
+  <script defer src="/shared/js/lazy_loader.js"></script>
+  <script defer src="/shared/js/contact_photo_helper.js"></script>
+  <script defer src="/shared/js/fb/fb_request.js"></script>
+  <script defer src="/shared/js/fb/fb_data_reader.js"></script>
+  <script defer src="/shared/js/fb/fb_reader_utils.js"></script>
+  <script defer src="/shared/js/event_dispatcher.js"></script>
+  <script defer src="/views/shared/js/navigation.js"></script>
+  <script defer src="/views/shared/js/inter_instance_event_dispatcher.js"></script>
+  <script defer src="/views/shared/js/time_headers.js"></script>
+  <script defer src="/views/shared/js/contacts.js"></script>
+  <script defer src="/services/js/drafts.js"></script>
+  <script defer src="/services/js/threads.js"></script>
+  <script defer src="/views/shared/js/utils.js"></script>
+  <script defer src="/views/shared/js/settings.js"></script>
+  <script defer src="/services/js/message_manager.js"></script>
+  <script defer src="/views/shared/js/app.js"></script>
+  <script defer src="/services/js/lib/bridge.js"></script>
+  <script defer src="/services/js/activity/client.js"></script>
+  <script defer src="/services/js/activity/service.js"></script>
+  <script defer src="/shared/js/multi_sim_action_button.js"></script>
+  <script defer src="/views/shared/js/task_runner.js"></script>
+  <script defer src="/views/shared/js/wbmp.js"></script>
+  <script defer src="/views/shared/js/smil.js"></script>
+  <script defer src="/views/shared/js/shared_components.js"></script>
+  <script defer src="/views/conversation/js/link_helper.js"></script>
+  <script defer src="/shared/js/image_utils.js"></script>
+  <script defer src="/views/conversation/js/attachment.js"></script>
+  <script defer src="/views/conversation/js/attachment_renderer.js"></script>
+  <script defer src="/views/conversation/js/subject_composer.js"></script>
+  <script defer src="/views/conversation/js/compose.js"></script>
+  <script defer src="/shared/js/gesture_detector.js"></script>
+  <script defer src="/lib/bridge.js"></script>
+  <script defer src="/services/js/activity/activity_client.js"></script>
+  <script defer src="/services/js/activity/activity_service.js"></script>
```

Comment:
Forgot to update ```service``` to ```shim``` here, but maybe we don't need neither ```shim``` nor ```activity_handler``` here at all, since nobody uses it.